README.md

```diff
@@ -1,10 +1,63 @@
 ## angular-filemanager
 File manager developed with AngularJS and Bootstrap by [Jonas Sciangula Street](https://github.com/joni2back)
 
+### use in your project
+
+#### 1) install and use
+1) bower install --save angular-filemanager
+2) include the dependencies in your project:
+
+    <!-- third party -->
+    <script src="/bower_components/angular-translate/angular-translate.min.js"></script>
+    <script src="/bower_components/angular-cookies/angular-cookies.min.js"></script>
+    
+    <!-- angular-filemanager -->
+    <link rel="stylesheet" href="/bower_components/angular-filemanager/dist/angular-filemanager.css">
+    <script src="/bower_components/angular-filemanager/dist/angular-filemanager.min.js"></script>
```

Comment:
When you use bower, bower_components is the standard folder for third party dependecies. So, I suggest to use the standard.